models/multiphase/d2q9_pf/Dynamics.R

```diff
@@ -27,36 +27,37 @@ AddDensity( name="h[6]", dx=-1, dy= 1, group="h")
 AddDensity( name="h[7]", dx=-1, dy=-1, group="h")
 AddDensity( name="h[8]", dx= 1, dy=-1, group="h")
 
-AddQuantity(name="Normal",unit="1/m",vector=T)
-AddQuantity(name="PhaseField",unit="1")
-
-AddSetting(name="IntWidth", default=0.33333, comment='Interface width')
-AddSetting(name="Mobility", default=0.001, comment='Mobility')
-AddSetting(name="PhaseField", 
-           default=0.5, 
-           comment='Phase Field marker scalar', 
-           zonal=T
-           )
-
-
-
-AddSetting(name="OverwriteVelocityField", default="0")
-#########################################################
-
-
-# THIS QUANTITIES ARE NEEDED FOR PYTHON INTEGRATION EXAMPLE
+# THIS DENSITIES ARE NEEDED FOR PYTHON INTEGRATION EXAMPLE
 # COMMENT OUT FOR PERFORMANCE
```

Comment:
Can we have a `BC` option here also?

Reply:
I've added no_bc instead  - tests and examples rely on the presence of bc. This is a demonstrator for "advection" model, so injecting velocity filed makes a lot of sense here